src/sentry/integrations/slack/utils/notifications.py

```diff
@@ -34,16 +40,16 @@ def send_incident_alert_notification(
         # Integration removed, but rule is still active.
         return False
 
+    organization = incident.organization
     chart_url = None
-    if features.has("organizations:metric-alert-chartcuterie", incident.organization):
```

Comment:
@sentaur-athena removing this is causing tests to break and have errors :(
Cool if we tackle this after?